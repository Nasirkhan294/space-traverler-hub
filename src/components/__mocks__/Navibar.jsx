import React from 'react';

const Navibar = () => (
  <div>
    <h1>Space mission hub</h1>
    <ul>
      <li>
        <a href="/">Rockest</a>
      </li>
      <li>
        <a href="/missions">Missions</a>
      </li>
      <li>
        <a href="/my-profile">My Profile</a>
      </li>
    </ul>
  </div>
);

export default Navibar;
