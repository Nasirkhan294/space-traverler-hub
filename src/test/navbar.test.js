import React from 'react';
import renderer from 'react-test-renderer';
import Navibar from '../components/__mocks__/Navibar';

describe('Navibar', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Navibar />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
