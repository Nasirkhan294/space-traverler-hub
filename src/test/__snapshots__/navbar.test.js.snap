// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navibar renders correctly 1`] = `
<div>
  <h1>
    Space mission hub
  </h1>
  <ul>
    <li>
      <a
        href="/"
      >
        Rockest
      </a>
    </li>
    <li>
      <a
        href="/missions"
      >
        Missions
      </a>
    </li>
    <li>
      <a
        href="/my-profile"
      >
        My Profile
      </a>
    </li>
  </ul>
</div>
`;
